services/supervisord/supervisord.go

```diff
@@ -679,4 +680,42 @@ stdout_logfile_maxbytes = 10MB
 stdout_logfile_backups = 3
 redirect_stderr = true
 {{end}}
+
+{{define "grafana"}}
```

Comment:
And how grafana was started before?

Reply:
It's probably some merge conflict, this item was removed recenlty by @nikita-b

Reply:
Grafana was started by https://github.com/percona/pmm-server/blob/main/supervisord.conf#L60-L84

Reply:
There is a PR that removes it https://github.com/percona/pmm-server/pull/318